openassessment/xblock/static/js/src/lms/oa_staff_info.js

```diff
@@ -53,12 +53,52 @@ OpenAssessment.StaffInfoView.prototype = {
             function(html) {
                 // Load the HTML and install event handlers
                 $('#openassessment__student-info', view.element).replaceWith(html);
+                view.ScoreOverrideHandler();
             }
         ).fail(function(errMsg) {
                 view.showLoadError('student_info');
             });
     },
+    
+    /**
+     Upon request, submits an override grade.
+     **/
+    peerScoreOverride: function() {
+    	var sel = $('#openassessment__staff-info', this.element);
+    	var student_id = sel.find('#openassessment__student_id').val();
+    	var points_override = sel.find('#openassessment_override_score').val();
+    	var points_possible = sel.find('#openassessment_points_possible').html();
+    	this.server.peerScoreOverride(student_id, points_override, points_possible).done(
+            function(points_override) {
+                $('#openassessment_points_override', this.element).html(points_override);
+            }
+        ).fail(function(errMsg) {
+                $('#openassessment_points_override', this.element).html(errMsg);
+        });
+    },
+    
+    /**
+     Handler for Peer score override.
+     This must be run after loadStudentInfo has completed so that
+     the required HTML elements are present.
+     **/
+    ScoreOverrideHandler: function() {
```

Comment:
This name should start with a lower-cased letter.

On more of a nit level: I find that method names like "verbNoun" make it easier to differentiate between methods and properties/accessors.

```javascript
function handleScoreOverride() {}
function overridePeerScore() {}
```